src/markdown-pages/docs/ecosystem/ecosystem-installing.md

```diff
@@ -0,0 +1,84 @@
+---
+path: "/docs/ecosystem/installing"
+title: "Installing the Galasa Ecosystem in Docker"
+---
+
+The following section explains how to bring up a Galasa Ecosystem on a Docker server by using a Docker Operator. Currently the Docker Operator only supports the amd64 platform. A s390x (zLinux) implementation is planned for a future release.
+
+The instructions assume a basic understanding of how Docker works.
+
+## Before you begin
+
+To deploy a Galasa ecosystem using the Docker Operator, you need a Docker Engine, preferably on a Linux server, but it can be run on a laptop. If you are using a Mac or Windows OS, you  need Docker Desktop running a "socat" container.
```

Comment:
"can be run on a laptop for experimental reasons." We dont want people actually hosting them locally